### DISCUSSION TITLE
Increase the width of chunk output

### DISCUSSION THREAD

**Discussion by elgabbas** [quarto-dev/quarto-cli#3934] [Q&A] [answered]
Hello,

I would like to increase the area occupied with the output of the R chunk. For example, using this simple code returns the following output:
  
```{r}
print(cbind(iris, iris))
```

![image](https://user-images.githubusercontent.com/7375051/211927321-567dcce0-5bd5-410d-a9c4-952dbe25dfcf.png)

I want to increase the number of characters that fit each line; i.e., allow the filling of the empty area to the right.

Any advice? 
Thanks.

**Comment by mcanouil** [comment, marked as answer]
This is tied to `getOption("width")`, so set your desired width with `options(width = XXXX)`.

**Reply by elgabbas** [reply]
This was a rather basic R question! I expected that this has something specific to quarto.... Thanks...

**Reply by mcanouil** [reply]
FYI, same applies to `rmarkdown` (both are using `knitr`).

**Reply by cbrnr** [reply]
In case people are wondering how to set the output width globally, you can include these lines in the YAML header:

```
engine: knitr
knitr:
  opts_chunk: 
    R.options:
      width: 120
```
